### ISSUE TITLE
Is it possible to have a destructor-like native function be called when object is released?

### ISSUE THREAD

**Issue by treasurebox** [jxcore/jxcore#604]
If I am wrapping a native object with ```JX_WrapObject()```, it seems that in most of the cases it would be a good idea to register a function which is called when given object is gc'd. Is this possible? Thanks!

**Comment by obastemur** [comment]
Yes its possible but we need to make that implementation public. Can be part of 0.3.1.0 too

**Comment by treasurebox** [comment]
Cool!
